### DISCUSSION TITLE
Audio also needs to be optimized

### DISCUSSION THREAD

**Discussion by wlioi** [ClassicOldSong/Apollo#50] [Ideas]
1.Windows volume Settings do not control the volume of the projection device 
2. Turning on the console and the projection device at the same time will sound at the same time and feel an echo because of the delay

**Comment by ClassicOldSong** [comment]
I don't understand what you're talking about clearly, would you please translate using ChatGPT or just post in your native language?

Audio part do have confusions in various aspects and they need to be redesigned, but the redesign needs to be based on some other enhancements I have planned. So it's not very doable right now. Would you try tweaking the config in Audio/Video tab and see if they can somewhat improve your experience?
